lib/tasks/hydranorth-solr.rake

```diff
@@ -6,6 +6,29 @@ require 'rdf/turtle'
 namespace :hydranorth do
   namespace :solr do
 
+    desc "Index objects by batch files"
```

Comment:
Do you have a sample file to test this with? Does this belong in the README as well? Is this in any way related to ```rake batch:ingest_csv[file,tmp_dir,mode]                   # batch ingest from a csv file - used by ERA Admin and ERA Assistants```?